package.json

```diff
@@ -261,6 +261,18 @@
           }
         }
       }
+    ],
+    "languages": [
+      {
+        "id": "blend",
+        "extensions": [".blend"],
+        "aliases": ["Blend File", "blend"],
+        "icon": {
+          "light": "./icons/blender_icon.ico",
+          "dark": "./icons/blender_icon.ico"
+        },
+        "filenames": ["blender.exe", "blender-launcher.exe"]
```

Comment:
Linux and Mac do not have exe extensions 😋
And I guess I will not torture you with #160

Reply:
Added variants without extensions 😺